internal/provider/provider.go

```diff
@@ -77,6 +92,39 @@ func (p *infisicalProvider) Schema(ctx context.Context, _ provider.SchemaRequest
 				Sensitive:   true,
 				Description: "Machine identity client secret. Used to fetch/modify secrets for a given project",
 			},
+			"auth": schema.SingleNestedAttribute{
+				Optional:    true,
+				Description: "The configuration values for authentication",
+				Attributes: map[string]schema.Attribute{
+					"universal": schema.SingleNestedAttribute{
+						Optional:    true,
+						Description: "The configuration values for Universal Auth",
+						Attributes: map[string]schema.Attribute{
+							"client_id": schema.StringAttribute{
+								Optional:    true,
+								Sensitive:   true,
+								Description: "Machine identity client ID. Used to fetch/modify secrets for a given project",
+							},
+							"client_secret": schema.StringAttribute{
+								Optional:    true,
+								Sensitive:   true,
+								Description: "Machine identity client secret. Used to fetch/modify secrets for a given project",
+							},
+						},
+					},
+					"oidc": schema.SingleNestedAttribute{
+						Optional:    true,
+						Description: "The configuration values for OIDC Auth",
+						Attributes: map[string]schema.Attribute{
+							"identity_id": schema.StringAttribute{
+								Optional:    true,
```

Comment:
Should `identity_id` be `optional: true`?

Reply:
same here